docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -143,18 +101,24 @@ You may need to perform some additional configuration before you are able to run
 Installing Ansible
 ------------------
 
-Use ``pip`` in your selected Python environment to install the Ansible package of your choice for the current user:
+Use ``pip`` in your selected Python environment to install the minimal ``ansible-core`` package for the current user:
 
 .. code-block:: console
 
-    $ python3 -m pip install --user ansible
+    $ python3 -m pip install --user ansible-core
 
-Alternately, you can install a specific version of ``ansible-core`` in this Python environment:
+You can install a specific version of ``ansible-core`` if you need:
```

Comment:
not sure I agree that this change makes it more readable. 'if you need' is an odd expression to end the sentence with. The original "Alternately" clearly indicates this is optional and an alternate to the prior install step.

Reply:
Changed as it was, thanks for explaining